test/testMdastToVDOM.js

```diff
@@ -79,7 +79,7 @@ describe('Test MDAST to VDOM Transformation', () => {
     assertTransformerYieldsDocument(
       transformer, `
       <div>
-        <a name="h1"></a>
+        <a name="user-content-h1"></a>
```

Comment:
is this relevant to the XSS changes?

Reply:
Same as above. This is the DOM clobbering escape